pkg/termios/termios_linux.go

```diff
@@ -19,8 +19,8 @@ type TTY struct {
 	f *os.File
 }
 
-func New() (*TTY, error) {
-	f, err := os.OpenFile("/dev/tty", os.O_RDWR, 0)
+func New(device string) (*TTY, error) {
```

Comment:
In most cases, "/dev/tty" is used and I'd rather not scatter "/dev/tty" all over the place. 
Could you name this func NewWithName or NewTty or some similar name and have New() just call it with "/dev/tty"? That way New() still covers the 7 out of 8 uses of the termios packages I know, but you can still cover the one case. Were you to do that, the impact of this improvement would be limited to just 2 files, not 7. This will also break external users of this package (e.g. cpu) so I'd rather see a change that does not do that.